static/js/publisher/form/fileInput.js

```diff
@@ -26,9 +37,32 @@ class FileInput extends React.Component {
       this.input.accept = restrictions.accept;
     }
     this.input.hidden = true;
+    this.input.multiple = true;
```

Comment:
This makes it possible to choose multiple files when clicking "Add" as well.

While being a nice UX improvement it has some drawbacks:
- it makes it quite easy to choose many images at once and therefor reach the limit of 5 causing warnings
- if single input contains multiple files they cannot be removed one by one (we cannot remove one file from input and keep others) - this can lead to some bugs or unexpected behaviour when input containing multiple files changes value
- and because it's not possible to remove files from input partially if many files are chosen and added but only few kept (rest removed) all of them will be uploaded anyway (from single input)

Reply:
> This makes it possible to choose multiple files when clicking "Add" as well.
> 
> While being a nice UX improvement it has some drawbacks:
> 
>     * it makes it quite easy to choose many images at once and therefor reach the limit of 5 causing warnings
> 
The way I implemented it you do get the "you have over 5 images" warning and the save button stays disabled, until you're at 5 images. I think this is ok, but would be good to have some UX feedback on it.

>     * if single input contains multiple files they cannot be removed one by one (we cannot remove one file from input and keep others) - this can lead to some bugs or unexpected behaviour when input containing multiple files changes value
> 
- Because the view is rendered from the state, you can remove 1-by-1 visually and....

>     * and because it's not possible to remove files from input partially if many files are chosen and added but only few kept (rest removed) all of them will be uploaded anyway (from single input)
> 
- ...only the images that get sent in the images array get uploaded to the store by the backend, so even if there are multiple images only those that are desired will be uploaded to the store (though yes, they'll be uploaded to our backend, then disregarded).

Reply:
It does cause issues.

Not sure about specific test case, but if you drag multiple files in, remove some (to make space for more), drag some more, remove some, drag, etc. Not all of screenshots are saved.

Reply:
I'm afraid we may need to have to limit screenshots interactions to one file until we switch from using file inputs and keep file objects via File API in state (and POST form via ajax instead of HTML).